examples/making-an-away-command.md

```diff
@@ -0,0 +1,114 @@
+# Making An Away Command

```

Comment:
This is more of a QOL thing, but with the way the command stands currently, it's not compatible immediately with [Guidebot](https://github.com/AnIdiotsGuide/guidebot). I'm aware it's not necessarily supposed to be drag and drop, but it might also be nice if there was a version that would work immediately for those using Guidebot.

Reply:
is guidebot in master?

Reply:
Guidebot is not, it's on 11.3. I don't see anything that is v12 specific.

Reply:
Also, with your whole reason for making this command, it never actually lets a person know if the user they mentioned is away. The command is seemingly redundant with Discord's away status.

Reply:
discord's away system doesn't display reason

Reply:
Yeah, it's less than ideal, but with the way the command is currently made, it is basically doing the exact same thing that Discord's away status does. Would you mind if I DM'd you on Discord to talk more about this?

Reply:
sure

Reply:
To be honest, this should move to a different PR.